charts/artifactory-ha/v2.5.3/questions.yml

```diff
@@ -258,6 +258,14 @@ questions:
   group: "Database Settings"
   show_subquestion_if: "true"
   subquestions:
+  - variable: postgresql.postgresqlPassword
+    default: ""
+    description: "PostgreSQL password"
+    type: string
```

Comment:
Using `type: password` might be better suited for these as it will hide the password in the text field and accomplish the same results.

Reply:
Thank you for your review. I have updated all passwords to type "password".